linode/manager
# fix: [M3-8578] - Fix console error on Volume Create - Linode Config selection

## Description 📝
The console Warning: Cannot update a component (VolumeCreate) while rendering a different component (Unknown), usually occurs when a state update is triggered during the rendering phase of another component. In the `ConfigSelect` component, this issue likely arises from the logic that automatically sets a new value. So, moving that logic to `useEffect` hook avoids triggering state updates during the render phase.


## Target release date 🗓️
9/30



## How to test 🧪

### Reproduction steps
(How to reproduce the issue, if applicable)
- Navigate to Volume Create Flow, Select linode and notice warning in the console.

### Verification steps
(How to verify changes)
- Checkout the branch and run in local
-  Navigate to Volume Create Flow, select linode and observe no console warning.

## As an Author I have considered 🤔

*Check all that apply*
